docs/product-assembly_dm-on-openshift/src/main/asciidoc/dm-openshift-prepare-con.adoc

```diff
@@ -1,10 +1,15 @@
 [id='dm-openshift-prepare-con']
 = Preparing to deploy {PRODUCT} in your OpenShift environment
 
-Before deploying {PRODUCT} in your OpenShift environment, you need to complete several preparatory tasks. You do not need to repeat these tasks if you want to deploy additional images, for example, for new versions of decision services or for other decision services. 
+Before deploying {PRODUCT} in your OpenShift environment, you need to complete several preparatory tasks. You do not need to repeat these tasks if you want to deploy additional images, for example, for new versions of 
+ifdef::DM[decision services or for other decision services]
+ifdef::PAM[processes or for other processes.]
+ 
 
 .Prerequisites
 
-. Ensure that at least four gigabytes of memory are available in the OpenShift environment.
-. Create the OpenShift project for the deployment. 
-. Log in to the project using the OpenShift web console and using the `oc` command.
\ No newline at end of file
+. At least four gigabytes of memory are available in the OpenShift environment.
+. The OpenShift project for the deployment is created.
+. If you intend to scale any of the {KIE_SERVER} pods, your OpenShift environment supports persistent volumes with ReadWriteMany mode.
+IMPORTANT: ReadWriteMany mode is not supported on OpenShift Online and OpenShift Dedicated.   
+. You have loggeg in to the project using the OpenShift web console and using the `oc` command.
```

Comment:
Prereqs should be bullets, not numbers. Search all and address any other instances. Also, correct "loggeg". And the IMPORTANT note is mis-formatted. Should be
```
* Line of prereq text.
+
IMPORTANT: Text of important thing.
* Next bulletted prereq.
```